src/mscorlib/shared/System/ArgumentException.cs

```diff
@@ -64,13 +64,12 @@ public ArgumentException(String message, String paramName)
         protected ArgumentException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            _paramName = info.GetString("ParamName");
+            throw new PlatformNotSupportedException();
```

Comment:
Do we want this method to call the base method instead of throwing up?

Besides, just a heads-up, this might be going to break [DCS_ArgumentExceptionObject](https://github.com/dotnet/corefx/blob/178a982df2f210b615e21af7b0920f0fe5425142/src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs#L1454).